static-site/tsconfig.json

```diff
@@ -61,7 +61,6 @@ be set and will rewrite this file if they aren't set.²
     ]
   },
   "include": [
-    ".next/types/**/*.ts",
```

Comment:
I guess this is unnecessary to include Next.js' internal type defs?

Reply:
I'm not sure what it's used for but it does seem like Next.js wants it there and running `next lint` will add it back. I've added it back in 78d8b9a6accbed2502241071c3ddd674ce72b6b7.

Maybe we should be running `next lint` instead of ESLint on static-site?

Reply:
> running `next lint` will add it back

next.js has a habit of silently modifying configs, e.g. cc763094726d7ea6501f8087da026d880fddd3e1